### DISCUSSION TITLE
Degree symbol display in a LabelControl

### DISCUSSION THREAD

**Discussion by Warren-Whitfield** [pelicanmapping/osgearth#2532] [Support Q&A] [answered]
So I have this code extract:
```c++
QChar degree = (DEGREE_SYMBOL); //0x00b0
QString qstr = degree;
system.decimalDegreesToDMS(coords.y(), D, M, S);
std::stringstream latstream;
latstream << std::fixed << std::setw(2) << std::setfill('0') << (D < 0 ? -D : D) << qstr.toStdString()
				         << std::fixed << std::setw(2) << std::setfill('0') << (M < 0 ? -M : M) << "'"
				         << std::fixed << std::setw(2) << std::setprecision(0) << std::setfill('0') << (S < 0 ? -S : S)
				         << (D < 0 ? "'' S" : "'' N");
std::string strText = "Latitude / Longitude\n" + latstream.str() + " " + lonstream.str() + "\nRange / Azimuth\n" + rngstream.str() + " " + azstream.str();
m_labelControl->setText(strText);
```

But when it ouputs the degree symbol it looks like this...

![image](https://github.com/gwaldron/osgearth/assets/159110675/b92a28e2-406f-4798-8807-94e56d80b9d0)

So what am I missing to get the degree symbol displayed correctly? In normal Qt widgets it displays fine without that 'A'.

**Comment by gwaldron** [comment, marked as answer]
Try simply outputting "\xb0" in your stream; this has worked for me in the past.

(osgEarth also has a `LatLongFormatter` utility class that may or may not be useful to you.)

**Comment by Warren-Whitfield** [comment]
Genius!
Works like a charm. I was obviously overthinking it, thanks Glenn 
![image](https://github.com/gwaldron/osgearth/assets/159110675/a1cc1397-4e64-4fb5-a958-f963fcab5af6)
